docs/userGuide/syntax/popovers.md

```diff
@@ -45,14 +45,25 @@
   </popover>
 </p>
 <h4 class="no-index">Content using slot</h4>
-<popover header="**Emoji header** :rocket:">
-  <div slot="content">
-    This is a long content...
-  </div>
-  <button class="btn btn-secondary">Hover</button>
-</popover>
-<br />
-<br />
+<p>
+  <popover effect="scale" header="**Emoji header** :rocket:">
+    <div slot="content">
+      This is a long content...
+    </div>
+    <button class="btn btn-secondary">Hover</button>
+  </popover>
+</p>
```

Comment:
This change, while good, is a bit unrelated as it is not within the scope of our issue (as this is not regarding to the `src` attribute)

Reply:
I see! I'll revert it